tsconfig.json

```diff
@@ -18,6 +18,8 @@
     "baseUrl": ".",
     "types": [
       "webpack-env",
+      "jest",
+      "node"
       // TODO: Add these back when we add unit testing
       // "mocha",
       // "chai"
```

Comment:
Don't these need to be added then?

Reply:
oh, we're not using mocha or chai, I can remove these commented out lines now. Good catch